packages/fxa-payments-server/src/components/CouponForm/index.tsx

```diff
@@ -222,22 +230,25 @@ export const CouponForm = ({
             </Localized>
           </div>
 
-          <button
-            name="apply"
-            className="button"
-            type="submit"
-            data-testid="coupon-button"
-            disabled={checkingCoupon || readOnly || subscriptionInProgress}
-          >
-            <Localized id="coupon-submit">
-              <span>Apply</span>
-            </Localized>
-          </button>
+          <div className="w-28 ml-4">
```

Comment:
Maybe same here. Hardcoding container to 112px wide with the following potential translations/strings:
```sh
 1. coupon-submit = Застосувати
 2. coupon-submit = వర్తింపజేయి
 3. coupon-submit = Appliquer
 4. coupon-submit = Applitgar
 5. coupon-submit = Toepassen
 6. coupon-submit = Прилагане
 7. coupon-submit = Применить
 8. coupon-submit = Alkalmaz
 9. coupon-submit = Anwenden
10. coupon-submit = Applicar
…
32. coupon-submit = Apply
…
41. coupon-submit = 적용
```

```js
// From within the cloned "fxa-content-server-l10n` repo folder.
const cp = require("child_process");

const QUERY = "coupon-submit =";
const res = cp.execSync(`git grep -h "${QUERY}" | sort | uniq`)
  .toString()
  .trim();

const lines = res.split("\n")
  .sort((a, b) => b.length - a.length);
console.log(lines.map((val, idx) => `${String(idx + 1).padStart(2, " ")}. ${val}`).join("\n"));
```

Reply:
Same here (more details in other comment). Thank you for pointing this out!